pillarbox-ui/src/main/java/ch/srgssr/pillarbox/ui/extension/ComposablePlayer.kt

```diff
@@ -168,6 +168,7 @@ fun Player.playerErrorAsState(): State<PlaybackException?> {
  * @return A [State] that represents whether shuffle mode is currently enabled.
  */
 @Composable
+@Deprecated("Use `ShuffleButtonState` from Media3's ui-compose instead.")
```

Comment:
Note: I've deprecated these two methods, but they don't seem to be used outside `pillarbox-android` in the SRGSSR mmz-srf organisations.

Reply:
If not used maybe it is easier to remove them.